spec/features/super_admin/upload_download_mou_spec.rb

```diff
@@ -27,14 +40,15 @@
     end
   end
 
-  context "when dowloading the MOU template" do
+  context "when downloading the MOU template" do
     before do
       attach_file("unsigned_document", Rails.root + "spec/fixtures/mou.pdf")
       click_on "Upload"
       click_on "Download current template"
     end
 
     it "downloads the MOU" do
+      convert_pdf_to_page
```

Comment:
I would just assert that the `page.current_path` has changed to `rails/active_storage/disk` as this will effectively point to the backing local URL for that asset. You can also check that it contains the file's name, but anything past that seems like overkill.